mcc/odin/odin/odin.go

```diff
@@ -0,0 +1,70 @@
+package odin
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// Init initializes connection to AWS API
+func Init() rdsiface.RDSAPI {
+	region := "us-east-1"
+	sess := session.New(&aws.Config{Region: aws.String(region)})
+	return rds.New(sess)
+}
+
+var duration = time.Duration(5) * time.Second
+
+// CreateDBInstance creates a new RDS database instance. If a vpcid is specified the security
+// group will be in that VPC
+func CreateDBInstance(
+	instanceName string,
+	instanceType string,
+	masterUser string,
+	masterUserPassword string,
+	size int64,
+	svc rdsiface.RDSAPI,
+) (result string, err error) {
+	params := &rds.CreateDBInstanceInput{
+		AllocatedStorage:     &size,
+		DBInstanceClass:      aws.String(instanceType),
```

Comment:
These being variables passed can be captured directly with `&instanceType` and `&instanceName`